desci-server/src/services/index.ts

```diff
@@ -9,3 +10,8 @@ export const crossRefClient = new CrossRefClient(
   '', // process.env.CROSSREF_API_KEY,
   process.env.CROSSREF_EMAIL,
 );
+export const metadataClient = new AutomatedMetadataClient(
+  // 'process.env.AUTOMATED_METADATA_API',
+  'http://host.docker.internal:5005',
```

Comment:
this should be loaded from an ENV var

Reply:
where is the docker compose update?

Reply:
For the automated metadata repo? https://github.com/desci-labs/automating-metadata-v1